### ISSUE TITLE
Support expanding GitHub permalinks

### ISSUE THREAD

**Issue by abitrolly** [forem/forem#6777]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when links to code are not expanded. If I paste the next link on a separate line to Discourse, it will get expanded with the source code section.

https://github.com/wagoodman/dive/blob/c6bb2372a8be13a80a81c00c04f9724970c0ae60/dive/image/image.go#L7-L10

Using the `github` block doesn't work either.

{% github https://github.com/wagoodman/dive/blob/c6bb2372a8be13a80a81c00c04f9724970c0ae60/dive/image/image.go#L7-L10 %}

**Describe the solution you'd like**

Just pasting https://github.com/wagoodman/dive/blob/c6bb2372a8be13a80a81c00c04f9724970c0ae60/dive/image/image.go#L7-L10 on a separate line is ideal case.

**Additional context**

Example rendering.

https://meta.discourse.org/t/discourse-github/99895/10

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @abitrolly, thanks for this feature request.

I wonder if the easiest solution might be to simply expand the `github` tag capabilities so that it also supports that.

**Comment by cmgorton** [comment]
Hello! Thanks for your contributions here.  I am going to close this issue.  We have a new process moving forward to address feature requests.
TLDR: we have new internal RFC process for feature requests. Any features that are more substantial and require more thought and detail will need to be discussed in [forem.dev](forem.dev).

You can read more about our Internal RFC Process and forem.dev Discussions in [this post](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4).

**Comment by abitrolly** [comment]
If the process is internal I guess it doesn't make sense to read about it given that addressing existing requests is not part of the process. Whatever you choose, it your project, so good luck with it.

**Comment by cmgorton** [comment]
Hey @abitrolly only the RFC process is currently internal but we are still openly championing request to create RFCs. Like many open source projects we are moving towards using Github for bug reports and other platforms for feature requests. This makes it less intimidating for contributors who come to our repository and see 100s of open issues. 
We are still very much listening and taking requests into consideration. I think this request is a good one and would likely get a lot of buy in if it were posted on forem.dev. I have already added a similar request to our internal review process that expands the functionality of our GitHub liquid tag. 
You can see a few examples of good conversations happening already on forem.dev and those requests finally getting priority.
Hope you have a good weekend. Let me know if you want to post this on forem.dev and need any help.

**Comment by abitrolly** [comment]
Auto posting original request and cross-posting here would be fine. I would prefer dealing with it the way `discorse` and  GitHub does - without additional tags.
